CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Snyk Security Changelog
+## [2.6.3]
+- Add the Issue View Options panel to the Snyk Security Settings.
+
```

Comment:
Thank you 🙏 

I have added the my missing change here https://github.com/snyk/vscode-extension/pull/457/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR2-R6

Is this a valid approach?

Reply:
We both need to update this version here so it's newer that the latest release (`2.7.0`) I believe